036-BGP/Coach/README.md

```diff
@@ -0,0 +1,57 @@
+
+# Networking Hackathon - Info for coaches

```

Comment:
Title of coach guide should be identical to student guide: "What The Hack - Using BGP for Hybrid Connectivity"